src/core/java/com/enderio/core/client/gui/screen/EIOScreen.java

```diff
@@ -101,6 +101,26 @@ public boolean mouseClicked(double pMouseX, double pMouseY, int pButton) {
         return super.mouseClicked(pMouseX, pMouseY, pButton);
     }
 
+    @Override
+    public boolean mouseDragged(double pMouseX, double pMouseY, int pButton, double pDragX, double pDragY) {
+        for (GuiEventListener widget : children()) {
+            if (widget instanceof AbstractWidget abstractWidget && abstractWidget.isActive()) {
```

Comment:
This breaks since the new widget doesn't actually get focus

Reply:
okay, maybe still change it so that it returns true on the first widget returning true, meaning it handled the dragging? So no slotdragging would occur or similar